improve parser error

I don't understand anything

```
--- FAIL: TestParser (0.00s)
    --- FAIL: TestParser/simple_select (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000014cc0], want &{Expressions:[0xc0000286b0] Table:0xc0000286c0 Where:<nil>}
    --- FAIL: TestParser/select_multiple_columns (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000014d80], want &{Expressions:[0xc0000286e0 0xc0000286f0] Table:0xc000028700 Where:<nil>}
    --- FAIL: TestParser/select_1; (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000014e40], want &{Expressions:[0xc000012598] Table:<nil> Where:<nil>}
    --- FAIL: TestParser/select_null (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000014f00], want &{Expressions:[0x6b5f40] Table:<nil> Where:<nil>}
    --- FAIL: TestParser/select_bool (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000014fc0], want &{Expressions:[0xc0000125a0] Table:<nil> Where:<nil>}
    --- FAIL: TestParser/select_bool_case_insensitive (0.00s)
        parser_test.go:131: Parser.Parse() = [0xc000015080], want &{Expressions:[0xc0000125a1] Table:<nil> Where:<nil>}
```

--- a/parser/parser_test.go
+++ b/parser/parser_test.go
@@ -138,19 +138,19 @@ func TestParser(t *testing.T) {
 			got, err := p.Parse()
 			if tt.wantErr == nil {
 				if err != nil {
-					t.Errorf("Praser.Parse() error = %+v, at position %d, want %+v", err, p.pos, tt.want)
+					t.Errorf("Praser.Parse() error = %v, at position %d, want %v", err, p.pos, tt.want)
 					return
 				}
 				if !reflect.DeepEqual(got, tt.want) {
-					t.Errorf("Parser.Parse() = %+v, want %+v", got, tt.want)
+					t.Errorf("Parser.Parse() = %v, want %v", got, tt.want)
 				}
 			} else {
 				if err == nil {
-					t.Errorf("Parser.Parse() = %+v, wantErr %+v", got, tt.wantErr)
+					t.Errorf("Parser.Parse() = %v, wantErr %v", got, tt.wantErr)
 					return
 				}
 				if !errors.Is(err, tt.wantErr) {
-					t.Errorf("Parser.Parse() error = %+v, at position %d, wantErr %+v", err, p.pos, tt.wantErr)
+					t.Errorf("Parser.Parse() error = %v, at position %d, wantErr %v", err, p.pos, tt.wantErr)
 				}
 			}
 		})

